Sources/SwiftUILogger/Internal/SearchBar.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  SearchBar.swift
+//  
+//
+//  Created by Ahmed Shendy on 12/11/2022.
+//
+
+import SwiftUI
+
+struct SearchBar: View {
+    
+    private var searchText: Binding<String>
+    private let placeholder: String
+    
+    init(searchText: Binding<String>, placeholder: String) {
+        self.searchText = searchText
```

Comment:
```suggestion
        self_.searchText = searchText
```

Reply:
@0xLeif Oh this is new to me, thanks for these changes.